src/components/dropdown/dropdown.svelte

```diff
@@ -38,6 +38,10 @@
   let button: HTMLButtonElement
   let dropdown: HTMLDivElement
 
+  export const close = () => {
+    isOpen = false
```

Comment:
probably still need to emit the close event? Not 100% sure about whether consumers should be able to cancel it though :D